Composer/packages/server/src/models/bot/sampler/BootstrapSampler.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { BootstrapSampler } from '@microsoft/bf-dispatcher/lib/mathematics/sampler/BootstrapSampler';
+
+const SAMPLE_SIZE_CONFIGURATION = 2;
+
+export interface IUtterance {
+  text: string;
+  intent: string;
+  entities: any[];
+}
+
+export class ComposerBootstrapSampler extends BootstrapSampler<number> {
+  private _times = 10;
```

Comment:
this ratio is also a key indicator, which should come through config, this is probably a ratio we adjust frequently